src/Symfony/Component/HttpFoundation/ResponseHeaderBag.php

```diff
@@ -30,7 +30,7 @@ class ResponseHeaderBag extends HeaderBag
     protected $computedCacheControl = array();
 
     /**
-     * @var array
+     * @var Cookie[]
```

Comment:
this is wrong. It is not an array of Cookie objects. It is an array of arrays of arrays of Cookie objects